core/src/test/java/com/linecorp/armeria/common/stream/PathStreamMessageTest.java

```diff
@@ -49,11 +49,15 @@
 
 class PathStreamMessageTest {
 
+    private static final Path testFilePath = Paths.get(
+            "src/test/resources/testing/core",
+            PathStreamMessageTest.class.getSimpleName(),
+            "test.txt");
+
     @ArgumentsSource(SubscriptionOptionsProvider.class)
     @ParameterizedTest
     void readFile(SubscriptionOption[] options) {
-        final Path path = Paths.get("src/test/resources/com/linecorp/armeria/common/stream/test.txt");
```

Comment:
We had a lot of constant duplication in this class. It could have been nicer if this was addressed in the review phase.